### DISCUSSION TITLE
MViewer & Postgis

### DISCUSSION THREAD

**Discussion by cotegeo** [mviewer/mviewer#783] [Q&A]
Bonjour,

Je (re)découvre MViewer, et je voudrais savoir s'il est possible d'avoir des couches PostGreSQL/PostGIS, voire des tuiles vectorielles. En effet, je travaille avec des API qui proposent la génération de tuiles vectorielles.

Merci pour vos retours

Sylvain BECQUAERT

**Comment by lecault** [comment]
Bonjour,

Alors on ne peut pas charger directement une couche provenant de PostGresql/PostGIS. Après, il est peut être possible de charger des données via une API et le type customlayers dans mviewer. La communauté mviewer est intéréssé si des tests sont fait sur l'exploitation d'API.

Concernant les tuiles vectorielles, le type vector-tile tms a été implémenté mais juste pour les données provenant de l'IGN : https://github.com/mviewer/mviewer/issues/675 mais sommes intéréssé par l'implémentation de toutes tuiles vectorielles.

Bonne journée,
Loïc Ecault Région Bretagne

**Comment by tbrunelle9** [comment]
Bonjour, 

Il est possible en effet de passer par un customlayers pour charger des couches via du JavaScript et notamment du PHP pour se connecter à une base de données Postgre/Postgis. 

Bonne journée,
Tom Brunelle, OPenIG
